daemon/cmd/daemon_main.go

```diff
@@ -1127,6 +1127,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.StringSlice(option.NodeLabels, []string{}, "List of label prefixes used to determine identity of a node (used only when enable-node-selector-labels is enabled)")
 	option.BindEnv(vp, option.NodeLabels)
 
+	flags.Bool(option.EnableServiceProtocolDifferentiation, true, "TODO")
```

Comment:
I'm not familiar with cilium config options so take this comment with that in mind.
I wonder if you want to make it the negative `--disable-service-protocol-awareness` with default `false`, users will automatically pick this value without the need of setting the flag

Reply:
> users will automatically pick this value without the need of setting the flag

that's already the case thanks to that second parameter set to `true`, which is the default value for the flag :+1:

Reply:
I see, it always pick the default you set from the flag